data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3433 @@
+					goto leave
+				`	"Why can you not just use a jump drive and go there yourself?"`
+				`	"What will my cut be?"`
+					goto cut
+				`	"Sorry I will not risk my ship by going anywhere close to these pirates."`
+					decline
+			`	"Jump drives are a very prized possession among my people, I would not risk one by travelling trough hostile space. You had access to a jump drive so you should know about the hostiles to the South. The wanderers would not be very happy to see me either..." He seems disappointed by you having to ask such an obvious thing.`
+			choice
+				`	"Let's go then!"`
+					goto leave
+				`	"Sorry I will not risk my ship by going anywhere close to these pirates."`
+					decline
+			label cut
+			`	"You humans always seem obsessed with money... It will be a lot... really a lot! I will only take 25% of it, and you know how much my people value jump drives."`
+			label leave
+			`	"One last thing: We should not go there by jump drive, especially considering where we are going, it would only attract unwanted attention to us. It would not be of use anyway considering we should not travel trough wanderer space, I do not want to try their patience or tolerance to seeing me on your ship." You acompagny him to your ship, and he admits to quite like the interior of it, as well as general look of it, this ship "is adequate for a warrior," he says.`
+				accept
+	on visit
+		dialog `You do not have the jump drives. You either did not go to <destination> to get the jump drives yet, or left your escort allocated to carrying the jump drives behind, in which case better launch and wait for it.`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	to complete
+		has "Unfettered: Jump Drive Retrieval: offered"
+	on complete
+		outfit "Jump Drive" -4
+		payment 3000000
+		log `Helped a young Unfettered captain to bring jump drives from a pirate world. It was a really... unforgettable experience, I almost died multiple times but in the end I saved him.`
+		log "People" "Si-Ra" `A young impetuous Unfettered captain who seems to grow wiser by the day. He has impressive leading and combat skill, and seems to be a fan of getting bigger and bigger ship. He always leaves room for improving tactics.`
+		dialog `"Here is your share : <payment>. You have saved my life, captain <first>, and I shall never forget it. You have earned the right to know my name : "Si-Ra". Come see me again soon if you are up for a challenge, I am sure that I will have what you want." Then he leaves, after doing what you assume to be a salute to you, his fist risen. You are taken by surprise by the act, and before you have the time to react, he is already leaving. He is probably in a hurry to buy that new ship he told you about.`
+
+
+
+mission "Unfettered: Attracted Attention"
+	invisible
+	landing
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Prove your strength 2: active"
+	on offer
+		require "Jump Drive" 1
```

Comment:
I tested it, it does not; because the other mission needs to be active and that arrives as you land
Conditions are not refreshed until you depart again